contributed_definitions/nyaml/NXphysical_process.yaml

```diff
@@ -0,0 +1,12 @@
+category: base
+doc: |
+  A planned or unplanned process which results in physical changes in a specified material.
+  Examples include sample preparation, material transformation, or (partially) destrustive measurement.
+type: group
+NXphysical_process(NXobject):
+  date(NX_DATE_TIME):

```

Comment:
Strongly against having this only rather add also start_time and end_time to enable an interval in EOSC space-time

Reply:
Agreed and fixed in commit [b2521bb](https://github.com/FAIRmat-NFDI/nexus_definitions/pull/56/commits/b2521bbc81396f2c8d8ac4a24b7ab6c8005b82e8)